indico/modules/events/editing/operations.py

```diff
@@ -247,16 +247,31 @@ def undo_review(revision):
 
 
 @no_autoflush
-def update_review_comment(revision, text):
+def update_review(revision, text, tags):
     _ensure_revision_can_be_updated(revision)
-    changes = {'comment': (revision.comment, text)}
-    log_fields = {'comment': 'Comment'}
-    revision.comment = text
+
+    changes = {}
```

Comment:
I think you can do `changes = revision.populate_from_dict(data)` here and it'll do all the logic below for you.